### ISSUE TITLE
fix hamburger nav position and overflow

### ISSUE THREAD

**Issue by chrisvaillancourt** [chrisvaillancourt/website#24]
Opening the nav hamburger on small viewports shows a space for the scrollbar. 

Additionally, opening the hamburger triggers horizontal overflow.

<img width="281" alt="image" src="https://github.com/chrisvaillancourt/website/assets/44307925/bfe32b10-5b09-473c-a2fb-78181560efae">

**Comment by chrisvaillancourt** [comment]
No longer relevant after #73.
